pkg/term/term_windows.go

```diff
@@ -29,53 +31,98 @@ func StdStreams() (stdIn io.ReadCloser, stdOut, stdErr io.Writer) {
 		return os.Stdin, os.Stdout, os.Stderr
 	case os.Getenv("MSYSTEM") != "":
 		// MSYS (mingw) does not emulate ANSI well.
-		return winconsole.WinConsoleStreams()
+		return windows.ConsoleStreams()
 	default:
-		return winconsole.WinConsoleStreams()
+		return windows.ConsoleStreams()
 	}
 }
 
 // GetFdInfo returns file descriptor and bool indicating whether the file is a terminal.
 func GetFdInfo(in interface{}) (uintptr, bool) {
-	return winconsole.GetHandleInfo(in)
+	return windows.GetHandleInfo(in)
 }
 
 // GetWinsize retrieves the window size of the terminal connected to the passed file descriptor.
 func GetWinsize(fd uintptr) (*Winsize, error) {
-	info, err := winconsole.GetConsoleScreenBufferInfo(fd)
+
+	info, err := windows.GetConsoleScreenBufferInfo(fd)
+	if err != nil {
+		return nil, err
+	}
+
+	font, err := windows.GetCurrentConsoleFont(fd)
 	if err != nil {
 		return nil, err
 	}
 
-	// TODO(azlinux): Set the pixel width / height of the console (currently unused by any caller)
-	return &Winsize{
+	winsize := &Winsize{
 		Width:  uint16(info.Window.Right - info.Window.Left + 1),
 		Height: uint16(info.Window.Bottom - info.Window.Top + 1),
-		x:      0,
-		y:      0}, nil
+		x:      uint16(font.FontSize.X),
```

Comment:
The corresponding Linux winsize struct uses uint16 (see http://lxr.linux.no/linux+v3.19.1/arch/alpha/include/uapi/asm/termios.h#L33). We followed the same for consistency. Ok?